environment.yml

```diff
@@ -1,78 +1,79 @@
 name: xclim
 channels:
-    - conda-forge
-    - defaults
+  - numba  # Added to gain access to Python3.12-compatible numba release candidates.
+  - conda-forge
+  - defaults
 dependencies:
-    - python >=3.8
-    - astroid
-    - boltons >=20.1
-    - bottleneck >=1.3.1
-    - cf_xarray >=0.6.1
-    - cftime >=1.4.1
-    - Click >=8.1
-    - dask >=2.6.0
-    - importlib-resources  # For Python3.8
-    - jsonpickle
-    - lmoments3
-    - numba
-    - numpy >=1.16
-    - pandas >=0.23,<2.2
-    - pint >=0.9
-    - poppler >=0.67
-    - pyyaml
-    - scikit-learn >=0.21.3
-    - scipy >=1.2
-    - statsmodels
-    - xarray >=2022.06.0,<2023.11.0
-    - yamale
-    # Extras
-    - eofs
-    - flox
+  - python >=3.8,<3.13
```

Comment:
Is there a reason to pin python < 3.13 here? Since we added `numba` to the deps, we've always been "limited" by them and we always had time between a new python release and updating our workflows.

Reply:
I have a tendency to want to place upper pins since it helps speed up the resolver on ReadTheDocs. Also, early versions of the next point release (3.13 here) tend to cause issues. I don't mind removing the pin here.